slackminion/dispatcher.py

```diff
@@ -139,6 +139,8 @@ def _parse_message(self, message):
         if message:
             try:
                 args = unicodedata.normalize('NFKD', message.text).split(' ')
+                # Remove extra spaces
+                args = [arg for arg in args if arg != '']
```

Comment:
Would it be better to just use string.split() here without any arguments?

Reply:
yes, good idea